.pre-commit-config.yaml

```diff
@@ -21,47 +21,43 @@ repos:
         exclude: '\.ipynb$'
       - id: mixed-line-ending
         args: [--fix=lf]
-  - repo: https://github.com/gitleaks/gitleaks
```

Comment:
Found gitleaks a bit painful to work with and I'm not too worried about secrets being accidentally added to this repo